shared/templates/file_groupowner/bash.template

```diff
@@ -4,6 +4,22 @@
 # complexity = low
 # disruption = low
 
+{{%- set ns=namespace(find_groups="") %}}
+{{%- set GROUPS=GID_OR_NAME.split("|") %}}
+{{%- for grp in GROUPS %}}
+{{%- set ns.find_groups=ns.find_groups ~ " ! -group " ~ grp %}}
+{{%- if loop.first %}}
+if getent group "{{{grp}}}" >/dev/null 2>&1; then
+  newgroup="{{{grp}}}"
+{{%- else %}}
+elif getent group "{{{grp}}}" >/dev/null 2>&1; then
+  newgroup="{{{grp}}}"
+{{%- endif %}}
+{{%- if loop.last %}}
```

Comment:
I believe this saw the same issue as #13192

Reply:
Fixed